src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public enum Colors {

```

Comment:
```suggestion
public enum Color {

```
better use singular for enums